libgobuster/dir.go

```diff
@@ -108,16 +108,28 @@ func SetupDir(s *State) bool {
 	guid := uuid.Must(uuid.NewV4())
 	wildcardResp, _ := GoGet(s, s.Url, fmt.Sprintf("%s", guid), s.Cookies)
 
-	if s.StatusCodes.Contains(*wildcardResp) {
-		s.IsWildcard = true
-		fmt.Println("[-] Wildcard response found:", fmt.Sprintf("%s%s", s.Url, guid), "=>", *wildcardResp)
-		if !s.WildcardForced {
-			fmt.Println("[-] To force processing of Wildcard responses, specify the '-fw' switch.")
+	if s.NegStatusCodes.Stringify() == "" {
```

Comment:
It would probably be better to use `len`?

* https://godoc.org/golang.org/x/tools/container/intsets#Sparse.Len